src/components/searchfacetnavigation.main.jsx

```diff
@@ -81,6 +81,7 @@ class SearchFacetNavigationMain extends React.Component {
           return (
             <div className="list-group-item facet-value" key={chosen._description[0].value}>
               <button type="button" className="form-check-label chosen" onClick={() => this.handleFacetSelection(encodeURIComponent(chosen._selectaction[0].self.uri))}>
+                <span className="checkmark chosen" />
```

Comment:
I'm noticing duplicate clicks are required for selecting facets. Can you please check what might be causing it?
![Apr-28-2019 15-56-53](https://user-images.githubusercontent.com/39800563/56869411-56af0180-69ce-11e9-83ce-dee2d4a42ec3.gif)